x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeParseProcessor.java

```diff
@@ -98,6 +130,16 @@ public boolean equals(Object obj) {
         }
 
         DateTimeParseProcessor other = (DateTimeParseProcessor) obj;
-        return Objects.equals(left(), other.left()) && Objects.equals(right(), other.right());
+        return Objects.equals(parser, other.parser)
+            && Objects.equals(left(), other.left()) && Objects.equals(right(), other.right());
+    }
+
+    @Override
+    public String toString(){
```

Comment:
Is this used? If so it shouldn't contain only the parser.toString() but also left & and right.

Reply:
It's useless, I remove it.